### ISSUE TITLE
[discussion] Fused MLPs

### ISSUE THREAD

**Issue by vadimkantorov** [pytorch/pytorch#92259]
### 🚀 The feature, motivation and pitch

I just stumbled on https://twitter.com/DrJimFan/status/1615018393601716224, there is https://github.com/NVlabs/tiny-cuda-nn which fuses small MLPs for fast training and inference. Can Inductor / Triton generate similar PTX / memory placement?

It seems at least as a good comparison target for benchmark

**Comment by ngimel** [comment]
Yes triton can generate similar code, we haven't taught inductor to do it yet though.
